app/models/concerns/searchable.rb

```diff
@@ -1,75 +1,90 @@
+# Including classes should implement the following methods:
+#
+# searchable_body_field - name of field containing body text
+# searchable_metadata_fields - array of additional field names to query
+# searchable_filter - WHERE clause that filter the matched records, e.g. to those visible to end user
 module Searchable
   extend ActiveSupport::Concern
 
+  # Disable cop as want SQL to be formatted for readability. Its already a complex query and splitting
+  # into fragments will make it less readable.
+  #
+  # rubocop:disable Metrics/BlockLength
```

Comment:
put the disable comment on the next line to apply to the whole block?

Reply:
the disable only works in this position?